src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -734,10 +741,7 @@ private void visitFunctionParameters(FunctionType ftype) {
         if (param.isOptionalArg()) {
           emit("?");
         }
-        visitTypeDeclaration(param.getJSType());
-        if (param.isVarArgs()) {
-          emit("[]");
```

Comment:
because visitTypeDeclaration emits the colon, so if I need a paren between the colon and the start of the type expression, then I have to inline visitTypeDeclaration here instead of calling it